src/plugins/condition/components/ConditionSet.vue

```diff
@@ -63,6 +66,7 @@ export default {
     data() {
         return {
             currentConditionOutput: '',
+            defaultConditionOutput: 'No output available',
```

Comment:
Would this string ever be displayed? Why not an empty string or "Default"?

Reply:
This string should never be displayed. But I like it being something other Default because if it ever were displayed it would be easier to spot as something potentially being broken.

Reply:
Actually. I just found out it will be displayed if the user selects string output but then enters no string.

Reply:
Hmm... that seems wrong to me. That should ideally be validated somehow. 'No output available' would be misleading because there is output but it's unlabeled. Seems it would be better, short of validation, to just leave it as an empty string.

Reply:
+1 to @JoelMcKinnon 's comment.

Reply:
Resolving this with empty string solution for now.